README.md

```diff
@@ -39,6 +37,39 @@ jenkins:
           inhibitInferRootDN: false
           rootDN: "dc=acme,dc=org"
           server: "ldaps://ldap.acme.org:1636"
+
+  nodes:
+    - permanent:
+        name: "static-agent"
+        remoteFS: "/home/jenkins"
+        launcher:
+          jnlp:
+
+  slaveAgentPort: 50000
+  agentProtocols:
+    - "jnlp2"
+tool:
+  git:
+    installations:
+      - name: git
+        home: /usr/local/bin/git
+unclassified:
```

Comment:
Maybe we want to drop this "unclassified" section to get better look and feel. IIUC the smtpHost issue is fixed anyway, CC @alecharp

Reply:
I can do that. It's my first commit here so I just need confirmation from someone to remove whole `unclassifield` node :).

Reply:
+1 from me, it is just an example

Reply:
👍 we have a demo, so we can cleanup the example in the main README :)

Reply:
I think that a short sample is still useful. The size is yet to be determined, but we go in a right direction

Reply:
Removed.